packages/dropdownMenu/README.md

```diff
@@ -5,24 +5,33 @@ A dropdown toggles an overlay menu of actions. It appears as a result of a user
 Dropdown menus can be opened and navigated via cursor and keyboard.
 
 ## DropdownMenu triggers
+
 By default, use one of the dropdown button components (e.g.: `PrimaryDropdownButton`) as the dropdown trigger.
 
 If you're not using a button as your dropdown trigger, be sure whatever you use looks like something the user is supposed to interact with.
 
 ## Icons and avatars in items
+
 Dropdown menu items can display an icon or avatar before or after the menu item text. By default, the avatar or icon is sized at `iconSizeXs`.
 
-## Disabled menu items
-TODO: Collaborate with the design team to decide on some guidelines for disabled items. For example: when should we render a disabled item instead of not showing the item in the dropdown at all?
```

Comment:
I think for now we can keep it KISS:

In general, follow considerations from the interactive component state guidelines (we link to this) as to when to disable/enable/hide menu items.
Menu items typically follow same interaction considerations as buttons. 
If all menu items are disabled, then disable it's launchpoint and do not show the menu.